index.html

```diff
@@ -1,155 +1,246 @@
-            <div class="skill-box skills">
-                <img src="img/icon-skills.png" alt="Skills">
-                <h2>Skills</h2>
-                <ul>
-                    <li>Codekit</li>
-                    <li>GitHub</li>
-                    <li>Codepen</li>
-                    <li>Gitlab</li>
-                    <li>Terminal</li>
-                </ul>
+          </div>
+          <div class="work">
+            <div class="project-info">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavScript</li>
+                <li>html</li>
+              </ul>
+              <button>See Project</button>
             </div>
-        </section>
+          </div>
+          <div class="work">
+            <div class="project-info">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavScript</li>
+                <li>html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-section">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
+      </section>
+      <section class="skills-section">
+        <div class="skill-box language">
+          <img src="img/icon-languages.png" alt="Language" />
+          <h2>Language</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="skill-box framework">
+          <img src="img/icon-frameworks.png" alt="Framework" />
+          <h2>Frameworks</h2>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skill-box skills">
+          <img src="img/icon-skills.png" alt="Skills" />
+          <h2>Skills</h2>
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact-section">
+        <div class="contact">
+          <div class="text-contact">
+            <h3>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h3>
+          </div>
+          <div class="contact-form">
+            <form action="#" class="contact-form">
```

Comment:
- [ ] [REQUIRED] Good job so far! Form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). action value should be similar to https://formspree.io/f/{form_id} method value equals `POST`. Please implement my suggestion 🙏🏼